specification/dfp/resource-manager/Microsoft.DFP/preview/2021-02-01-preview/examples/createInstance.json

```diff
@@ -0,0 +1,45 @@
+{
+  "parameters": {
+    "subscriptionId": "613192d7-503f-477a-9cfe-4efc3ee2bd60",
+    "resourceGroupName": "TestRG",
+    "instanceName": "azsdktest",
+    "api-version": "2017-10-01",
+    "instanceParameters": {
+      "tags": {
+        "testKey": "testValue"
+      },
+      "location": "West US",
+      "properties": {
+        "administration": {
+          "members": [
+            "azsdktest@microsoft.com",
+            "azsdktest2@microsoft.com"
+          ]
+        }
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+        "id": "/subscriptions/613192d7-503f-477a-9cfe-4efc3ee2bd60/resourceGroups/TestRG/providers/Microsoft.DFP/instances/azsdktest",
+        "location": "West US",
+        "name": "azsdktest",
+        "properties": {
+          "administration": {
+            "members": [
+              "azsdktest@microsoft.com",
+              "azsdktest2@microsoft.com"
+            ]
+          },
+          "provisioningState": "Provisioning",
```

Comment:
for a non-terminal state, put should return 201

Reply:
fixed